libraries/AP_Proximity/AP_Proximity.h

```diff
@@ -173,9 +173,10 @@ class AP_Proximity
     bool valid_instance(uint8_t i) const;
 
     // parameters for all instances
-    AP_Int8 _raw_log_enable;                            // enable logging raw distances
+    AP_Int8 _raw_log_enable;                           // enable logging raw distances
     AP_Int8 _ign_gnd_enable;                           // true if land detection should be enabled
     AP_Float _filt_freq;                               // cutoff frequency for low pass filter
+    AP_Float _min_obs_alt_m;                           // Minimum altitude -in meters- below which proximity should not work.
```

Comment:
maybe name this variable _alt_min?  we don't actually need the _m because by default we use meters.

Reply:
done.